assets/dev/js/editor/document/ui-states/scrubbing-mode.js

```diff
@@ -0,0 +1,20 @@
+import UiStateBase from 'elementor-api/core/states/ui-state-base';
+
+export class ScrubbingMode extends UiStateBase {
+	getId() {
+		return 'scrubbing-mode';
+	}
+
+	getOptions() {
+		// When scrubbing mode is on, body gets class 'e-ui-state--document-scrubbing-mode__on'
```

Comment:
remove the comment

Reply:
The only way to understand this is to read the entire docs, I believe the code need to be understood quicker.